swift_browser_ui_frontend/src/pages/BrowserPage.vue

```diff
@@ -6,22 +6,18 @@
       :projects="projects"
     />
     <ProgressBar v-if="isUploading || isChunking" />
-    <div
-      class="breadcrumb"
-      aria-label="breadcrumbs"
-    >
-      <ul
-        id="breadcrumb-list"
-        style="margin-left:5%;margin-top:1%;"
+      <b-breadcrumb
+        style="margin-left:5%;margin-top:1%;font-size:1rem;"
       >
-        <BreadcrumbListElement
+        <b-breadcrumb-item
+          tag="router-link"
           v-for="item in getRouteAsList ()"
           :key="item.alias"
-          :alias="item.alias"
-          :address="item.address"
-        />
-      </ul>
-    </div>
+          :to="item.address"
+        >
+          {{ item.alias }}
```

Comment:
maybe missing `truncate(100)` so that we can avoid long names see #336  for details ?